modules/client-configuration/pages/supported-features-es.adoc

```diff
@@ -13,52 +13,52 @@ The icons in this table indicate:
+| Service pack migration | icon:times[role="danger"] | icon:times[role="danger"] | icon:times[role="danger"]
+| Basic Virtual Guest Management icon:asterisk[role="none"] | ``Traditional`` | icon:check[role="green"] | icon:check[role="green"]
+| Advanced Virtual Guest Management icon:asterisk[role="none"] | icon:times[role="danger"] | ``Salt`` | ``Salt``
+| Virtual Guest Installation (Kickstart), as Host OS | ``Traditional`` | ``Traditional`` | icon:times[role="danger"]
+| Virtual Guest Installation (image template), as Host OS | ``Traditional`` | icon:check[role="green"] | icon:check[role="green"]
+| System deployment (PXE/Kickstart) | icon:check[role="green"] | icon:check[role="green"] | icon:check[role="green"]
+| System redeployment (Kickstart) | ``Traditional`` | icon:check[role="green"] | icon:times[role="danger"]
 | Contact methods | ``Traditional``: OSAD, RHNSD, SSH-push. +
 ``Salt``: ZeroMQ, Salt-SSH | ``Traditional``: OSAD, RHNSD, SSH-push. +
-``Salt``: ZeroMQ, Salt-SSH
-| Works with {productname} Proxy | icon:check[role="green"] | icon:check[role="green"]
-| Action chains | icon:check[role="green"] | icon:check[role="green"]
-| Software crash reporting | icon:times[role="danger"] | ``Traditional``
-| Staging (pre-download of packages) | icon:check[role="green"] | icon:check[role="green"]
-| Duplicate package reporting | icon:check[role="green"] | icon:check[role="green"]
-| CVE auditing | icon:check[role="green"] | icon:check[role="green"]
-| SCAP auditing | icon:check[role="green"] | icon:check[role="green"]
-| Package verification | ``Traditional`` | ``Traditional``
-| Package locking | ``Traditional`` | ``Traditional``
-| System locking | ``Traditional`` | ``Traditional``
-| System snapshot | ``Traditional`` | ``Traditional``
-| Configuration file management | icon:check[role="green"] | icon:check[role="green"]
+``Salt``: ZeroMQ, Salt-SSH | ``Salt``: ZeroMQ, Salt-SSH
+| Works with {productname} Proxy | icon:check[role="green"] | icon:check[role="green"] | icon:check[role="green"]
+| Action chains | icon:check[role="green"] | icon:check[role="green"] | icon:check[role="green"]
+| Software crash reporting | icon:times[role="danger"] | ``Traditional`` | icon:times[role="danger"]
+| Staging (pre-download of packages) | icon:check[role="green"] | icon:check[role="green"] | icon:check[role="green"]
+| Duplicate package reporting | icon:check[role="green"] | icon:check[role="green"] | icon:check[role="green"]
+| CVE auditing | icon:check[role="green"] | icon:check[role="green"] | icon:check[role="green"]
+| SCAP auditing | icon:check[role="green"] | icon:check[role="green"] | icon:check[role="green"]
+| Package verification | ``Traditional`` | ``Traditional`` | icon:times[role="danger"]
+| Package locking | ``Traditional`` | ``Traditional`` | icon:times[role="danger"]
+| System locking | ``Traditional`` | ``Traditional`` | icon:times[role="danger"]
+| System snapshot | ``Traditional`` | ``Traditional`` | ``Traditional``
```

Comment:
The problem with ES8 is, that we only support "salt". There is no traditional stack. So saying `traditional` here is wrong.

We should in general think about what we do with these OSes which only support `salt`.
As documented [check] mean both. That means all `[check]` in ES8 needs to be changed to `salt`.